service/pebble/plan/doc.go

```diff
@@ -0,0 +1,7 @@
+// Copyright 2023 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// Package plan defines the various types to work with the Pebble plan and layers.
+//
+// This is a subset of the types in github.com/canonical/pebble/internal/plan.
```

Comment:
Maybe note that this was **copied** from the Pebble code?

Reply:
Noted